src/Bridge/Doctrine/Orm/Filter/NumericFilter.php

```diff
@@ -118,13 +121,27 @@ protected function filterProperty(string $property, $value, QueryBuilder $queryB
         $field = $property;
 
         if ($this->isPropertyNested($property, $resourceClass)) {
-            list($alias, $field) = $this->addJoinsForNestedProperty($property, $alias, $queryBuilder, $queryNameGenerator, $resourceClass);
+            list($alias, $field, $associations) = $this->addJoinsForNestedProperty($property, $alias, $queryBuilder, $queryNameGenerator, $resourceClass);
+            $metadata = $this->getNestedMetadata($resourceClass, $associations);
+        } else {
+            $metadata = $this->getClassMetadata($resourceClass);
         }
+
+        if (!isset(self::DOCTRINE_NUMERIC_TYPES[$metadata->getTypeOfField($field)])) {
+            $this->logger->notice('Invalid filter ignored', [
```

Comment:
Maybe should you log the filter name, the class and the property too (same in all logs).

Reply:
Those are logged in the exception.